comixed-web/src/app/library/components/comic-edit/comic-edit.component.ts

```diff
@@ -100,6 +101,29 @@ export class ComicEditComponent implements OnInit {
 
   ngOnInit(): void {}
 
+  onSaveDetails(): void {
+    this.confirmationService.confirm({
+      title: this.translateService.instant('comic.save-details.title'),
+      message: this.translateService.instant('comic.save-details.message'),
+      confirm: () => {
+        this.logger.trace('Saving the comic details');
+
+        this.store.dispatch(
+          saveComicDetails({
+            comic: {
+              ...this._comic,
+              publisher: this.comicForm.controls.publisher.value,
+              series: this.comicForm.controls.series.value,
+              volume: this.comicForm.controls.volume.value,
+              issueNumber: this.comicForm.controls.issueNumber.value
+            }
+          })
+        );
+        this.comicForm.markAsPristine();
```

Comment:
Let's remove this line here. The form should only be marked as pristine when the updated comic is received from the backend and the form is reloaded with the values received.